lib/improver/spotdata/tests/test_neighbour_finding.py

```diff
@@ -0,0 +1,742 @@
+# -*- coding: utf-8 -*-
```

Comment:
The major problem with these tests at the moment, similar to #14, is the lack of testing on anything other than the output from the `process` method of the `PointSelection` class. I think that it would be much better to have unit tests for the functions within `common_functions.py`, and therefore the number of tests purely on the `process` method would probably be greatly reduced. This would also make the tests much easier to follow, and it would make it much easier to assess what has been tested and what hasn't.